Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -930,23 +930,30 @@ private class MacroApplication<Context: MacroExpansionContext>: SyntaxRewriter {
   }
 
   override func visit(_ node: VariableDeclSyntax) -> DeclSyntax {
-    var node = super.visit(node).cast(VariableDeclSyntax.self)
+    var rewrittenNode = super.visit(node).cast(VariableDeclSyntax.self)
 
-    guard !macroAttributes(attachedTo: DeclSyntax(node), ofType: AccessorMacro.Type.self).isEmpty else {
-      return DeclSyntax(node)
+    guard !macroAttributes(attachedTo: DeclSyntax(rewrittenNode), ofType: AccessorMacro.Type.self).isEmpty else {
+      return DeclSyntax(rewrittenNode)
     }
 
-    guard node.bindings.count == 1,
-      var binding = node.bindings.first
+    guard rewrittenNode.bindings.count == 1,
+      var binding = rewrittenNode.bindings.first
     else {
-      contextGenerator(Syntax(node)).addDiagnostics(
+      contextGenerator(Syntax(rewrittenNode)).addDiagnostics(
```

Comment:
I think this should also be `contextGenerator(Syntax(node))`, no?

Reply:
Yes, it should. Thanks for catching it. Not sure how I missed it.